spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/LogFile.java

```diff
@@ -39,13 +40,27 @@
 	 * The name of the Spring property that contains the name of the log file. Names can
 	 * be an exact location or relative to the current directory.
 	 */
-	public static final String FILE_PROPERTY = "logging.file";
+	public static final String FILE_PROPERTY = "logging.file.name";
```

Comment:
I think this property should be deprecated in favour of `FILE_NAME_PROPERTY` (see below) using `@deprecated` in the javadoc and linking to the new constant. Its value should be left as `logging.file`.

Reply:
Done